### PR TITLE
Fix/issue 140, unauthenticated git://github.com/ when installing dependencies

### LINKED ISSUE #140: Unauthenticated git://github.com/ issue when installing dependencies
Same issue as reported on [flow_manager #35](https://github.com/kytos-ng/flow_manager/issues/35): 

```
❯ tox
GLOB sdist-make: /mnt/b/repos/kytos/setup.py
py36 create: /mnt/b/repos/kytos/.tox/py36
py36 installdeps: -rrequirements/dev.txt
ERROR: invocation failed (exit code 1), logfile: /mnt/b/repos/kytos/.tox/py36/log/py36-1.log
======================================================================================= log start ========================================================================================
Obtaining kytos-sphinx-theme from git+git://github.com/kytos/sphinx-theme.git#egg=kytos-sphinx-theme (from -r requirements/dev.txt (line 7))
  Cloning git://github.com/kytos/sphinx-theme.git to ./.tox/py36/src/kytos-sphinx-theme
  Running command git clone --filter=blob:none -q git://github.com/kytos/sphinx-theme.git /mnt/b/repos/kytos/.tox/py36/src/kytos-sphinx-theme
  fatal: remote error:
    The unauthenticated git protocol on port 9418 is no longer supported.
  Please see https://github.blog/2021-09-01-improving-git-protocol-security-github/ for more information.
```

### PR DESCRIPTION
**Pull request by viniarck**
Fixes #140 

### Description of the change

- It turns out unauthenticated URLs starting with git:// is no longer supported by GitHub, so I had to update the requirements file accordingly to a supported format. I hit this when I started to work on a environment from scratch.
- I also disabled the doc tests because of issue #141
- I've also bumped `click` to unblock `black` as a development dependency (since it was conflicting if any NApps were try to install it since `kytos` had a hard dependency on this `click==7.1.1` version). I noticed we still have several frozen deps from 2020, eventually, it might be worth doing a second pass trying to upgrade some of them, reviewing change logs to add important upstream fixes.

### CODE REVIEW

File: requirements/dev.in [hunk 1]
```diff
@@ -1,5 +1,5 @@
--e git+git://github.com/kytos/python-openflow.git#egg=python-openflow
--e git+git://github.com/kytos/sphinx-theme.git#egg=kytos-sphinx-theme
+-e git+https://github.com/kytos-ng/python-openflow.git#egg=python-openflow
+-e git+https://github.com/kytos/sphinx-theme.git#egg=kytos-sphinx-theme
```
Comment (italovalcy) [inline requirements/dev.in:2]:
```suggestion
-e git+https://github.com/kytos-ng/sphinx-theme.git#egg=kytos-sphinx-theme
```
I've just forked from the kytos project to kytos-ng

File: requirements/dev.txt [hunk 2]
```diff
@@ -4,8 +4,8 @@
 #
 #    pip-compile --output-file=requirements/dev.txt requirements/dev.in requirements/run.txt
 #
--e git+git://github.com/kytos/sphinx-theme.git#egg=kytos-sphinx-theme  # via -r requirements/dev.in
--e git+git://github.com/kytos/python-openflow.git#egg=python-openflow  # via -r requirements/dev.in, -r requirements/run.txt
+-e git+https://github.com/kytos/sphinx-theme.git#egg=kytos-sphinx-theme  # via -r requirements/dev.in
```
Comment (italovalcy) [inline requirements/dev.txt:7]:
```suggestion
-e git+https://github.com/kytos-ng/sphinx-theme.git#egg=kytos-sphinx-theme  # via -r requirements/dev.in
```

**Review by italovalcy — CHANGES_REQUESTED** [review]
The PR is very good, thanks @viniarck. I've just suggested two changes on the Kytos Sphinx Theme (which was not forked yet, and I just forked to kytos-ng)

**Comment by viniarck** [comment]
> The PR is very good, thanks @viniarck. I've just suggested two changes on the Kytos Sphinx Theme (which was not forked yet, and I just forked to kytos-ng)

Awesome. @italovalcy thanks for catching this, and forking the repo. I've just committed the pushed the suggested changes.

**Review by italovalcy — APPROVED** [review]

**Pull request merged**